app/Models/Loggable.php

```diff
@@ -66,14 +73,40 @@ public function logCheckout($note, $target /* What are we checking out to? */)
             'log_id' => $log->id
         ];
 
-        if ($settings = Setting::getSettings()) {
-           // $settings->notify(new CheckoutNotification($params));
-        }
+        // Get the admin email address we should be sending notifications to, if any
+        $recipient = new \App\Models\Recipients\AdminRecipient();
+
+        // Send to the admin, if settings dictate
+        if (static::class == Asset::class) {
+
+            if ($settings->admin_cc_email!='') {
+                $recipient->notify(new CheckoutAssetNotification($params));
+            }
+
+            $target->notify(new CheckoutAssetNotification($params));
+        } elseif (static::class == Accessory::class) {
+
+            if ($settings->admin_cc_email!='') {
+                $recipient->notify(new CheckoutAccessoryNotification($params));
+            }
+
+            $target->notify(new CheckoutAccessoryNotification($params));
+        } elseif (static::class == Consumable::class) {
+
+            if ($settings->admin_cc_email!='') {
+                $recipient->notify(new CheckoutConsumableNotification($params));
+            }
 
-        if (method_exists($target, 'notify')) {
-            $target->notify(new CheckoutNotification($params));
+            $target->notify(new CheckoutConsumableNotification($params));
+        } elseif (static::class == LicenseSeat::class) {
+            if ($settings->admin_cc_email!='') {
+                $recipient->notify(new CheckoutLicenseNotification($params));
+            }
+
+            $target->notify(new CheckoutLicenseNotification($params));
         }
 
+
```

Comment:
Thoughts/opinions about changing this whole section to something like (untested) this? Less duplication, but maybe a little harder to grok at first staring...

```
        $checkoutClass = null;
        switch(static::class) {
            case Asset::class:
                $checkoutClass = CheckoutAssetNotification::class;
                break;
            case Accessory::class:
                $checkoutClass = CheckoutAccessoryNotification::class;
                break;
            case Consumable::class:
                $checkoutClass = CheckoutConsumableNotification::class;
                break;
            case LicenseSeat::class:
                $checkoutClass = CheckoutLicenseNotification::class;
                break;

        }
        if ($settings->admin_cc_email!='') {
            $recipient->notify(new $checkoutClass($params));
        }
        $target->notify(new $checkoutClass($params));
```